content/apps/managing-teammates.md

```diff
@@ -0,0 +1,24 @@
+---
+menu:
+  main:
+    parent: advanced
+title: Managing Teammates
+weight: 10
+---
+
+To add a new teammate to your organization:
+
+1. [Send them an invite](https://login.cloud.gov/invitations/new).
+1. Run
```

Comment:
For new users, there's an implied waiting period between these steps... There's no way to set a role on someone who hasn't yet confirmed their invitation. Probably best to insert something like *wait for confirmation that their account is set up* as a second step. It's pretty crappy UX in that way, but we have bigger UX fish to fry before we can tackle it (in the deck; can't do anything for the CLI).